source/production/F0.Minesweeper.Components/Abstractions/Enums/DifficultyLevel.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace F0.Minesweeper.Components.Abstractions.Enums
+{
+	public enum DifficultyLevel
+	{
+		[Display(Name = "Easy*")]
```

Comment:
Do we still need the `Display` attributes? I see that we now use `Enum.GetNames` for retrieving the display values

Reply:
I think I wanted to goof around to see if it will theoretically work with translations 
I don't think it works this way anyway :)